tornjak-backend/api/agent/server.go

```diff
@@ -625,7 +625,7 @@ func (s *Server) ConfigureDefaults() (error) {
 	expBackoff := backoff.NewExponentialBackOff()
 	expBackoff.MaxElapsedTime = time.Second
 
-	s.Db, err = agentdb.NewLocalSqliteDB("sqlite3", "./localtornjakdb", expBackoff)
+	s.Db, err = agentdb.NewLocalSqliteDB("sqlite3", "./tornjak.sqlite3", expBackoff)
```

Comment:
I think this should end up being the `run/spire/data/tornjak.sqlite3`
to make sure it is in the right persistent volume. Currently this is not testable because we made the DB a required configuration. 

Otherwise, the default value can be up to discussion - but given all our quickstarts have the same structure, we can have the default be placing it in the right volume for now